tests/js/common/shell/aql-index-usage.js

```diff
@@ -122,6 +124,54 @@ function IndexUsageSuite () {
       assertTrue(success > 0, success);
     },
 
+    
+    testPrimaryIndexLookupConsistency : function () {
+      debugSetFailAt("RocksDBCollection::read-delay");
```

Comment:
Just wondering if this will work in the cluster case...
IMO the failure point will only be set on the coordinator here, but have no effect on a DB server. 
It should be fine in single server though.